### PR TITLE
feat: add arrow thin style

### PR DESCRIPTION
**Pull request by RudyBlack**
## Details
I think need a thin arrow. So add a thin arrow style with class.

### CODE REVIEW

File: css/arrow.css [hunk 1]
```diff
@@ -86,3 +86,15 @@
...
+
+.flicking-arrow-prev.thin::before,
+.flicking-arrow-prev.thin::after,
+.flicking-arrow-next.thin::before,
+.flicking-arrow-next.thin::after{
+  height: 3px;
+}
+
+.flicking-arrow-prev.thin::after,
+.flicking-arrow-next.thin::after{
```
Comment (WoodNeck) [inline css/arrow.css:98]: As other modifier classes are using names like `is-xxx`, this should be also changed like that.

```suggestion
.flicking-arrow-prev.is-thin::before,
.flicking-arrow-prev.is-thin::after,
.flicking-arrow-next.is-thin::before,
.flicking-arrow-next.is-thin::after{
  height: 3px;
}

.flicking-arrow-prev.is-thin::after,
.flicking-arrow-next.is-thin::after{
```

File: css/arrow.css [hunk 2]
```diff
@@ -86,3 +86,15 @@
...
+  top : calc(50% - 2px);
```
Comment (WoodNeck) [inline css/arrow.css:99]: Please remove the space between `top` and `:`

```suggestion
  top: calc(50% - 2px);
```

**Review by WoodNeck — APPROVED** [review]

**Review by WoodNeck — CHANGES_REQUESTED** [review]
Thanks for the PR!
I think I can agree that we need more default styles for those plugins.
It would be nice for you to include how they actually look like.

image

**Review by WoodNeck — APPROVED** [review]

**Review by daybrush — APPROVED** [review]
LGTM

**Review by malangfox — APPROVED** [review]
LGTM

**Pull request merged**